static/index.html

```diff
@@ -0,0 +1,220 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <meta
+      name="viewport"
+      content="width=device-width, initial-scale=1, shrink-to-fit=no"
+    />
+
+    <title>Execution Plan: Clean Page</title>
+
+    <link href="css/bootstrap.css" rel="stylesheet" />
+    <link href="css/index.css" rel="stylesheet" />
+
+    <!--data-evolv-environment="619f15bb6a" - this is the mutually exclusive contexts-->
+    <!-- Go to / and the home context should match, confirmation events for 'Mutually Exclusive Contexts' project-->
+    <!-- Go to /product.html and the product context should match, no confirmation events should fire as it is a non entry page-->
+    <!-- Go to /dashboard and the dashboard context should match, confirmation events for 'Dashboard only' project-->
+
+    <!--data-evolv-environment="57ba3d9272" - this is the overlapping contexts-->
+    <!-- Go to / and the all context should match, confirmation events for 'With All Context' project-->
+    <!-- Go to /dashboard.html and the all context and dashboard should match, confirmation events for 'With All Context' and 'Dashboard only' projects-->
+
+    <script
+      src="https://media.evolv.ai/asset-manager/releases/next/webloader.js"
```

Comment:
This will need to use the parameters won''t they? In order to test the environment we want